synfig-studio/src/gui/app.h

```diff
@@ -353,13 +353,7 @@ class App : public Gtk::Application, private IconController
 	static const std::map<const char*, const char*>& get_default_accel_map();
 	static void load_file_window_size();
 	static void load_language_settings();
-	static void set_workspace_default();
-	static void set_workspace_compositing();
-	static void set_workspace_animating();
-	static void set_workspace_from_template(const std::string &tpl);
-	static void set_workspace_from_name(const std::string &name);
 	static void load_custom_workspaces();
```

Comment:
Maybe move `load_custom_workspaces()`, `edit_custom_workspace_list()` and `WorkspaceHandler *studio::App::workspaces` too?

Reply:
Done in a new commit